src/lib/marineCli.ts

```diff
@@ -62,16 +63,28 @@ export const initMarineCli = async (
     cwd,
     printOutput = true,
   }): Promise<string> => {
-    return execPromise({
-      command: marineCLIPath,
-      args,
-      flags,
-      spinnerMessage:
-        message === undefined
-          ? undefined
-          : getMessageWithKeyValuePairs(message, keyValuePairs),
-      options: { cwd },
-      printOutput,
-    });
+    try {
+      return await execPromise({
+        command: marineCLIPath,
+        args,
+        flags,
+        spinnerMessage:
+          message === undefined
+            ? undefined
+            : getMessageWithKeyValuePairs(message, keyValuePairs),
+        options: { cwd },
+        printOutput,
+      });
+    } catch (e) {
+      if (
+        isObject(e) &&
+        hasKey("message", e) &&
+        typeof e["message"] === "string"
+      ) {
+        return commandObj.error(e["message"]);
```

Comment:
`errorMessage`, `expectedErrorMessage`, `message` – so many names for a conceptually same field 🤔

Reply:
here is different. Error objects by convention have `message`. errorMessage and expectedErrorMessage is just a way to resolve promise and distinguish between the way it was resolved. I think we can avoid errorMessage, expectedErrorMessage and use instance of in execPromise instead, that would be better